Android/app/src/main/java/live/dittolive/chat/DittoHandler.kt

```diff
@@ -70,6 +71,12 @@ class DittoHandler {
                     // Disable sync with V3 Ditto
                     disableSyncWithV3()
                 }
+
+                // Set up Ditto Websocket URL
+                val transportConfig = DittoTransportConfig()
+                transportConfig.connect.websocketUrls.add(BuildConfig.DITTO_WEBSOCKET_URL)
+                ditto.transportConfig = transportConfig
```

Comment:
I'd use the `updateTransportConfig` convenience method here,

```suggestion
                ditto.updateTransportConfig {
                    it.connect.websocketUrls.add(BuildConfig.DITTO_WEBSOCKET_URL)
                }
```

It's not obvious or really documented very well, but instantiating a new instance of `DittoTransportConfig()` actually creates a new config with P2P transports disabled.

The convenience method takes the existing transport config and modifies it, then sets it correctly. The existing transport config, if you didn't make any modifications has P2P transports enabled by default.